randovania/games/samus_returns/logic_database/Area 2 - Dam Entryway.txt

```diff
@@ -59,11 +49,23 @@ Extra - asset_id: collision_camera
   * Extra - actor_name: LE_ChozoUnlockAreaDNA
   * Extra - actor_type: weightactivatedplatform
   * Extra - start_point_actor_name: ST_ChozoUnlockAreaDNA_Checkpoint
+  > Pickup (Missile Tank)
+      Any of the following:
+          Lightning Armor and Lay Any Bomb
+          All of the following:
+              # Damage through plants
+              Hazard Runs (Intermediate)
+              Any of the following:
+                  Hazard Damage ≥ 190 and Lay Power Bomb
+                  Hazard Damage ≥ 270 and Lay Bomb
   > Elevator to Area 3 - Factory Exterior
       All of the following:
           Morph Ball
-          # Paraby Swarm on the way
-          Dodge Enemies (Intermediate) or Defeat Swarms
+          Any of the following:
+              Wave Beam or After Area 2 (Dam Entryway) - Chozo Seal Grapple Block or Precise Aiming (Intermediate) or Lay Any Bomb
```

Comment:
I feel like Intermediate might be too high but it does help give the other options a chance.

Reply:
Actually, this was supposed to be beginner. mustve accidentally set it to int. instead